libcurl/_competitors.html

```diff
@@ -20,7 +20,7 @@
 <br><a href="../docs/comparison-table.html">FTP+HTTP Tools comparison</a>
 </div>
 <p>
-Free Software and Open Source projects have a long tradition of forks and
+Free- and Open Source Software projects have a long tradition of forks and
```

Comment:
FWIW, Wikipedia doesn't use the dash on Free so I think we should follow their lead: https://en.wikipedia.org/wiki/Free_and_open-source_software

Reply:
I agree with @danielgustafsson

Reply:
Ok, I will change. Sorry, I totally forgot this pull requests :P